assignments/callbacks.js

```diff
@@ -24,28 +24,60 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
 
+getLength(items, (length) => {
+  console.log(length);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[3]);
 }
 
+last(items, (lastItem) => {
+  console.log(lastItem);
+});
+
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x, y);
+}
+
+function add(x, y) {
+  return x + y;
 }
 
+console.log(sumNums(3, 5, add));
+
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x,y);
+}
+
+function multiply(x, y) {
+  return x * y;
 }
 
+console.log(multiplyNums(4, 4, multiply));
+
 function contains(item, list, cb) {
```

Comment:
Please let me know if you have questions about this. One way to do is to iterate the array, and check if item is in the list. If it is, pass true to the callback, else if it's not in the list, pass false to the callback.